### PR TITLE
APP-1328: Integrate `hAnalytics`

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: app/src/main/java/com/hedvig/app/ApplicationModule.kt [hunk 1]
```diff
@@ -618,6 +622,10 @@ val trackerModule = module {
     }
     single { MixpanelTracker(get()) } bind TrackerSink::class
     single { FirebaseTracker(get()) } bind TrackerSink::class
+    single<HAnalytics> { HAnalyticsFacade(getAll<HAnalyticsSink>().distinct()) }
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/ApplicationModule.kt:625]: I used the sources/sinks-based model for `hAnalytics` as well, in order to be able to build for example engineering tooling on top of it.

File: app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInTracker.kt [hunk 2]
```diff
@@ -2,13 +2,18 @@ package com.hedvig.app.feature.loggedin.ui
...
 
 class LoggedInTracker(
     private val trackingFacade: TrackingFacade,
-    private val screenTracker: ScreenTracker
+    private val screenTracker: ScreenTracker,
+    private val hAnalytics: HAnalytics,
 ) {
     fun setMemberId(memberId: String) = trackingFacade.identify(memberId)
     fun tabVisited(tab: LoggedInTabs) {
+        if (tab == LoggedInTabs.HOME) {
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInTracker.kt:14]: This is temporary, and the code will look a lot more like the statements below it once all events have been properly integrated, in APP-1341.

File: gradle/libs.versions.toml [hunk 3]
```diff
@@ -49,6 +49,7 @@ coroutines = "1.5.2"
 firebase = "28.4.2"
 flexbox = "3.0.0"
 fragmentViewBindingDelegate = "1.0.0"
+hAnalytics = "c2d3db94bc647ef6a8306e78d274525ae628b72f"
```
Comment (hedvigoscar) [inline gradle/libs.versions.toml:52]: Once `hAnalytics` stabilizes properly, I will move this from being a direct git hash reference to a release-number instead.

**Review by hugokallstrom — APPROVED** [review]
So good!

**Pull request merged**